ClientApp/src/components/reuseables/ProjectEvaluationResults/EvaluationNotesTable.tsx

```diff
@@ -1,18 +1,17 @@
-import { ProjectEvaluationEmployeeNote } from "placeholders";
 import React from "react";
 import Card from "react-bootstrap/Card";
 import Col from "react-bootstrap/Col";
 import ListGroup from "react-bootstrap/ListGroup";
 import Row from "react-bootstrap/Row";
-import { fullName } from "utilities";
+import { EvaluationNotes as EvaluationNote } from "_generated";
 
-interface EvaluationNotesProps {
-  notes: ProjectEvaluationEmployeeNote[];
+interface EvaluationNotesTableProps {
+  notes: EvaluationNote[] | undefined | null;
```

Comment:
To deal with nulls as little as possible, I would recommend
```suggestion
  notes: EvaluationNote[] | undefined;
```
Then when passing props, use `notes || undefined` to cast any falsy valye to `undefined`.

Reply:
Good call